### ISSUE TITLE
CMake Error at conf.d/config.cmake

### ISSUE THREAD

**Issue by Amit-Radur** [iotbzh/helloworld-html-application#3]
When I run cmake . . from build directory, I see following:

cmake ..
CMake Error at conf.d/config.cmake:163 (include):
  include could not find load file:

    CMakeAfbTemplates
Call Stack (most recent call first):
  CMakeLists.txt:22 (include)

-- Configuring incomplete, errors occurred!
See also "/home/amit/work/helloworld-html-application/CMakeFiles/CMakeOutput.log".

**Comment by harunobukurokawa** [comment]
I have same issue. So, I revert latest commit. https://github.com/iotbzh/helloworld-html-application/commit/f92a6dde4f18324cb5b866bd11094778541492a0
Then It works.

**Comment by claneys** [comment]
Yes, this is due to the migration of app-templates as a CMake module. Without the CMake module you can't build it anymore indeed :)
Please find the required module installation instructions [here](http://docs.automotivelinux.org/master/docs/devguides/en/dev/reference/cmakeafbtemplates/0_Abstract.html#installation-for-a-native-environment)
